app/src/main/java/com/zeapo/pwdstore/UserPreference.kt

```diff
@@ -374,6 +383,22 @@ class UserPreference : AppCompatActivity() {
             }
         }
 
+        private fun updateClearSavedPassphrasePrefs() {
+            clearSavedPassPreference?.apply {
+                val sshPass = encryptedPreferences.getString("ssh_key_local_passphrase", null)
+                val httpsPass = encryptedPreferences.getString("https_password", null)
+                if (sshPass == null && httpsPass == null) {
+                    isVisible = false
+                    return
+                }
+                title = when {
+                    httpsPass != null -> getString(R.string.clear_saved_passphrase_https)
+                    sshPass != null -> getString(R.string.clear_saved_passphrase_ssh)
+                    else -> null
+                }
+            }
```

Comment:
```suggestion
                }
                isVisible = true
            }
```
This might solve the issue ;-)